src/modules/feedback/components/__tests__/FeedbackBoxContainer.spec.js

```diff
@@ -356,13 +364,41 @@ describe('FeedbackBoxContainer', () => {
         extras={null}
         postFeedbackDataOverride={mockPostFeedbackData}
         theme={theme}
+        sendingStatus='SUCCESS'
         t={t} />
     )
 
     component.instance().handleSubmit()
     expect(mockPostFeedbackData).toHaveBeenCalledTimes(1)
   })
 
+  it('should not post data on submit failure', async () => {
+    const mockOnSubmit = jest.fn()
+    const component = shallow(
+      <FeedbackBoxContainer
+        location={location}
+        cities={cities}
+        isPositiveRatingSelected
+        onSubmit={mockOnSubmit}
+        closeFeedbackModal={() => {}}
+        extras={null}
+        theme={theme}
+        sendingStatus='ERROR'
+        t={t} />
+    )
+    // set as workaround to override: console error to log
+    const prevError = console.error
+    console.error = error => console.log(`Some expected error was thrown: ${error}`)
```

Comment:
@maxammann @klinzo can you tell me why we override the error stream in the `setupTest.js` like this
```
console.error = error => {
  throw Error(error)
}
```
It seems this only produces inconveniences.

Reply:
I don't know, not my code. I thought this was default behavior of jest to fail tests if something is logged to error... Thanks for pointing it out! @maxammann ?